### PR TITLE
lab 1 done

### PR DESCRIPTION
**Pull request by FrancFranc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<head>
+  <title> </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <main>
+    <div class="background">
```
Comment (traceyr) [inline index.html]: Just a suggestion, this could be a section tag instead of a div!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<head>
+  <title> </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <main>
```
Comment (traceyr) [inline index.html]: Looking at your wireframe, it seems like this main tag should actually be a header tag! Your main tag should be used for the main body of your website while your header should include images and your nav.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    </nav>
+
+  </main>
+
+
+
+<!-- Script Links -->
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.4/handlebars.js"></script>
+    <script type="text/javascript" src="labArray.js"></script>
```
Comment (Tenlia) [inline index.html]: labArray.js doesn't currently exist in your repo.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <script type="text/javascript" src="lab.js"></script>
```
Comment (Tenlia) [inline index.html]: lab.js doesn't currently exist in your repo, did you mean to import app.js?

**Pull request merged**